projects/plugins/classic-theme-helper-plugin/classic-theme-helper-plugin.php

```diff
@@ -41,44 +41,35 @@
 define( 'CLASSIC_THEME_HELPER_PLUGIN_URI', 'https://jetpack.com' );
 define( 'CLASSIC_THEME_HELPER_PLUGIN_FOLDER', dirname( plugin_basename( __FILE__ ) ) );
 
-// Add "Settings" link to plugins page.
-add_filter(
-	'plugin_action_links_' . CLASSIC_THEME_HELPER_PLUGIN_FOLDER . '/classic-theme-helper-plugin.php',
-	function ( $actions ) {
-		$settings_link = '<a href="' . esc_url( admin_url( 'admin.php?page=classic-theme-helper-plugin' ) ) . '">' . __( 'Settings', 'classic-theme-helper-plugin' ) . '</a>';
-		array_unshift( $actions, $settings_link );
+// Init Jetpack packages that are hooked into plugins_loaded.
+add_action( 'plugins_loaded', 'init_packages_plugins_loaded', 1 );
 
-		return $actions;
-	}
-);
-
-	// Init Jetpack packages that are hooked into plugins_loaded.
-	add_action( 'plugins_loaded', 'init_packages_plugins_loaded', 1 );
-
-	/**
-	 * Configure what Jetpack packages should get automatically initialized, using the plugins_loaded hook.
-	 *
-	 * @return void
-	 */
+/**
+ * Configure what Jetpack packages should get automatically initialized, using the plugins_loaded hook.
+ *
+ * @return void
+ */
 function init_packages_plugins_loaded() {
+	$jp_plugin_version = Constants::get_constant( 'JETPACK__VERSION' );
```

Comment:
In testing I get a fatal error - `PHP Fatal error: Uncaught Error: Class "Constants" not found (on line 53 of classic-theme-helper-plugin.php)` - same issue as Phan is reporting as well.

If I change this to:
```
if ( class_exists( 'Automattic\Jetpack\Constants')) {
	$jp_plugin_version = 'Automattic\Jetpack\Constants::get_constant( 'JETPACK__VERSION' );
}
```
... and the same in `init_packages_init`, then the fatal is prevented and I can see that the settings link is now gone, plus in testing I could confirm that Featured Content that I tested loaded via the plugin.

If I deactivate the Jetpack plugin, I also now get no errors, the features just don't load.